experimental/dds/tree2/src/feature-libraries/typed-schema/typedTreeSchema.ts

```diff
@@ -72,66 +65,24 @@ export type Unenforced<_DesiredExtendsConstraint> = unknown;
  * See note on SchemaBuilder.fieldRecursive.
  * @alpha
  */
-export class TreeNodeSchema<
-	Name extends string = string,
-	T extends Unenforced<TreeSchemaSpecification> = TreeSchemaSpecification,
-> {
-	// Allows reading fields through the normal map, but without losing type information.
-	public readonly objectNodeFields: ObjectToMap<
-		NormalizeObjectNodeFields<Assume<T, TreeSchemaSpecification>["objectNodeFields"]>,
-		FieldKey,
-		TreeFieldSchema
-	>;
-
-	public readonly objectNodeFieldsObject: NormalizeObjectNodeFields<
-		Assume<T, TreeSchemaSpecification>["objectNodeFields"]
-	>;
-
-	public readonly mapFields: WithDefault<
-		Assume<T, TreeSchemaSpecification>["mapFields"],
-		undefined
-	>;
-	// WithDefault is needed to convert unknown to undefined here (missing properties show up as unknown in types).
-	public readonly leafValue: WithDefault<
-		Assume<T, TreeSchemaSpecification>["leafValue"],
-		undefined
-	>;
-
-	public readonly name: Name & TreeNodeSchemaIdentifier;
-
-	public readonly info: Assume<T, TreeSchemaSpecification>;
+export abstract class TreeNodeSchemaBase<
+	const out Name extends string = string,
+	const out Specification = unknown,
+> implements TreeNodeStoredSchema
```

Comment:
It did so before, and removing it breaks some code. For example it would require a an API to convert to stored schema, and a few places to use it. Contextually typed data is one example, dynamic schema validation (once we have it) will be another place that can work with both, and would need a conversion if view schema don't implement stored.